Java/ds/PriorityQueueEg.java

```diff
@@ -0,0 +1,56 @@
+import java.util.Iterator;
+import java.util.PriorityQueue;
+import java.lang.*;
+import java.util.Scanner;
+
+//Priority Queue data structure is a special queue in which the elements are present not as per FIFO order but as per the natural elements or any xyz comparator used during queue creation.
+
+public class PriorityQueueEg {
+    
+    private static Scanner sc = new Scanner(System.in);
+
+    public static void main(String args[])
```

Comment:
Shift your core Code Logic into another function. Use main() just for your driver code.

Reply:
Oh yea...that way the code would look more readable.